### PR TITLE
Add option to only show in Wilderness

### PR DESCRIPTION
**Pull request by Jbleezy**
This makes it so the lines don't show when you are near the Wilderness but not over the ditch like at the GE.

### CODE REVIEW

File: src/main/java/at/nightfirec/wildernessmultilines/WildernessMultiLinesOverlay.java [hunk 1]
```diff
@@ -60,6 +61,12 @@ private WildernessMultiLinesOverlay(WildernessMultiLinesPlugin plugin, Wildernes
 	@Override
 	public Dimension render(Graphics2D graphics)
 	{
+		boolean inWilderness = client.getVarbitValue(Varbits.IN_WILDERNESS) == 1;
+		if (config.onlyShowInWilderness() && !inWilderness)
```
Comment (Nightfirecat) [inline src/main/java/at/nightfirec/wildernessmultilines/WildernessMultiLinesOverlay.java]: Var check before config check to reduce config hits
```suggestion
		final boolean inWilderness = client.getVarbitValue(Varbits.IN_WILDERNESS) == 1;
		if (!inWilderness && config.onlyShowInWilderness())
```

File: src/main/java/at/nightfirec/wildernessmultilines/WildernessMultiLinesConfig.java [hunk 2]
```diff
@@ -64,4 +67,15 @@ default Color spearLinesColor()
 	{
 		return Color.ORANGE;
 	}
+
+	@ConfigItem(
+			position = 4,
+			keyName = "onlyShowInWilderness",
+			name = "Only show in Wilderness",
+			description = "Only show the lines if you are in the Wilderness"
```
Comment (Nightfirecat) [inline src/main/java/at/nightfirec/wildernessmultilines/WildernessMultiLinesConfig.java]: Fix indentation
```suggestion
		position = 4,
		keyName = "onlyShowInWilderness",
		name = "Only show in Wilderness",
		description = "Only show the lines if you are in the Wilderness"
```

**Comment by Jbleezy** [comment]
Changes committed.

**Comment by Nightfirecat** [comment]
Sorry for the delay. I did see these changes, I've not gotten around to testing them just yet. I should have time to do that later today.

**Comment by Nightfirecat** [comment]
I've released this as v1.2.0 on the plugin hub in runelite/plugin-hub@93e1a6140268410f8515c59caa44158178fb32e4

**Pull request merged**